docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -47,55 +44,55 @@ mo --input_model tf_nasnet_large.onnx --layout "nhwc->nchw"
+In the first case, the Model Optimizer generates the IR with required preprocessing operations and no *`mean`* and *`scale`* parameters are required.
 
 In the second case, information about mean/scale values should be provided to the Model Optimizer to embed it to the generated IR.
-Model Optimizer provides command-line parameters to specify the values: `--mean_values`, `--scale_values`, `--scale`.
+Model Optimizer provides command-line parameters to specify the values: *`--mean_values`*, *`--scale_values`*, *`--scale`*.
 Using these parameters, Model Optimizer embeds the corresponding preprocessing block for mean-value normalization of the input data
-and optimizes this block so that the preprocessing takes negligible time for inference.
+and optimizes this block so that the preprocessing takes a little time for inference.
 
-For example, run the Model Optimizer for the PaddlePaddle* UNet model and apply mean-scale normalization to the input data.
+For example, run the Model Optimizer for the PaddlePaddle UNet model and apply mean-scale normalization to the input data.
 
 ```sh
 mo --input_model unet.pdmodel --mean_values [123,117,104] --scale 255
 ```
 
-## When to Reverse Input Channels <a name="when_to_reverse_input_channels"></a>
+## Reversing Input Channels <a name="when_to_reverse_input_channels"></a>
 Sometimes input images for your application can be of the RGB (BGR) format and the model is trained on images of the BGR (RGB) format,
 the opposite color channel order. In this case, it is important to preprocess the input images by reverting the color channels before inference.
-To embed this preprocessing step into IR, Model Optimizer provides the `--reverse_input_channels` command-line parameter to shuffle the color channels.
+To embed this preprocessing step into IR, Model Optimizer provides the *`--reverse_input_channels`* command-line parameter to shuffle the color channels.
 
-The `--reverse_input_channels` parameter applies to an input of the model in two cases.
+The *`--reverse_input_channels`* parameter applies to an input of the model in two cases.
  * Only one dimension in the input shape has a size equal to 3.
- * One dimension has an undefined size and is marked as `C` channel using `layout` parameters.
+ * One dimension has an undefined size and is marked as *`C`* channel using *`layout`* parameters.
 
-Using the `--reverse_input_channels` parameter, Model Optimizer embeds the corresponding preprocessing block for reverting
-the input data along channel dimension and optimizes this block so that the preprocessing takes negligible time for inference.
+Using the *`--reverse_input_channels`* parameter, Model Optimizer embeds the corresponding preprocessing block for reverting
+the input data along channel dimension and optimizes this block so that the preprocessing takes a little time for inference.
```

Comment:
```suggestion
the input data along channel dimension and optimizes this block so that the preprocessing takes only negligible time for inference.
```